services/federator/src/Federator/Run.hs

```diff
@@ -86,8 +87,8 @@ newEnv o _dnsResolver = do
   _applog <- LogExt.mkLogger (Opt.logLevel o) (Opt.logNetStrings o) (Opt.logFormat o)
   let _requestId = def
   let _runSettings = Opt.optSettings o
-  let _brig = mkEndpoint (Opt.brig o)
-  let _brigEndpoint = Opt.brig o
+  let _brig Brig = mkEndpoint (Opt.brig o)
+      _brig Galley = mkEndpoint (Opt.galley o)
```

Comment:
How about calling this something other than `_brig` as it is not about Brig only anymore? Perhaps `_componentEndpoint` or something like that?

Reply:
Good point, I forgot to name the field.